src/Octokit.Webhooks/Models/WorkflowRun.cs

```diff
@@ -91,5 +91,18 @@ public sealed record WorkflowRun
 
         [JsonPropertyName("workflow_url")]
         public string WorkflowUrl { get; init; } = null!;
+
+        [JsonPropertyName("actor")]
+        public User Actor { get; init; } = null!;
+
+        [JsonPropertyName("run_attempt")]
+        public long RunAttempt { get; init; }
+
+        [JsonPropertyName("run_started_at")]
+        [JsonConverter(typeof(DateTimeOffsetConverter))]
+        public DateTimeOffset RunStartedAt { get; init; }
+
```

Comment:
Missing `previous_attempt_url` in here[^1]. I think the definition is going to be:

```suggestion
[JsonPropertyName("previous_attempt_url")]
public string? PreviousAttemptUrl { get; init; };

```

[^1]: https://github.com/octokit/webhooks/blob/d93eebf4c48dc3c36f9c7ceffab356fa444bf9fc/payload-types/schema.d.ts#L6255-L6259

Reply:
Looks like I missed this one because it's a _Deployment_ WorkflowRun[^1], rather than a `WorkflowRun`, so that's why there wasn't a `previous_attempt_url` in the payload[^2]. Should I add a different type that's a subset, or leave it as is where various properties will just be `null` because they're absent?

[^1]: https://github.com/octokit/webhooks/blob/d93eebf4c48dc3c36f9c7ceffab356fa444bf9fc/payload-types/schema.d.ts#L2231
[^2]: https://github.com/octokit/webhooks/blob/d93eebf4c48dc3c36f9c7ceffab356fa444bf9fc/payload-types/schema.d.ts#L2156-L2159

Reply:
If `octokit/webhooks` splits these as two different models, then we should probably do so too.

`DeploymentWorkflowRun` is used in `DeploymentCreatedEvent` and `DeploymentStatusCreatedEvent`.
`WorkflowRun` is used in `WorkflowRunCompletedEvent` and `WorkflowRunRequestedEvent`

Reply:
Schema reference[^1]

[^1]: https://github.com/octokit/webhooks/blob/d93eebf4c48dc3c36f9c7ceffab356fa444bf9fc/payload-types/schema.d.ts#L6255-L6259

Reply:
I went through the schema definition and added the different models for the ones where they were either defined differently or inline as separate models/enums and rebased this onto #52.